spec/classes/foreman_proxy__plugin__ansible_spec.rb

```diff
@@ -18,6 +18,17 @@
         with_content(/:enabled: https/).
         with_content(%r{:ansible_dir: /etc/ansible})
     end
+
+    it 'should install ansible' do
+      should contain_package('ansible').with_ensure('present')
+    end
+
+    it 'should configure ansible.cfg' do
+      should contain_file('/usr/share/foreman-proxy/.ansible.cfg').
+        with_content(%r{default}).
+        with_content(%r{callback_whitelist = foreman}).
+        with_content(%r{local_tmp = /tmp})
+    end
   end
 
   describe 'with override parameters' do
```

Comment:
The override of `:working_dir` is no longer being tested as the default has changed to the overridden value, change the value here.